etc/adminhtml/system/adyen_security.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
```

Comment:
Are we going to re-create the whole section for this or can we put it in an already existing one?

Reply:
Hello @michaelpaul, thanks for the review. Default value is false for this configuration. Moving it to the `Required Settings` can lead the merchants to a wrong direction. Since, it is an optional security check, what do you think about keeping the current position?

Reply:
It's fine by me. We might need to update documentation pages as well.